tests/Fixer/Whitespace/NoTrailingWhitespaceFixerTest.php

```diff
@@ -96,6 +96,25 @@ public function provideFixCases()
             [
                 "<?php\necho <<<'EOT'\nInline Il y eut un   \r\nrire éclatant    \n     \n   \r\nEOT;\n\n",
             ],
+            [
+                "<?php\necho 'Hello World';",
+                "<?php \necho 'Hello World';",
+            ],
+            [
+                "<?php\r\necho 'Hello World';",
+                "<?php \r\necho 'Hello World';",
+            ],
+            [
+                "<?php\necho 'Hello World';",
+                "<?php  \necho 'Hello World';",
+            ],
+            [
+                "<?php\necho 'Hello World';",
+                "<?php	\necho 'Hello World';",
+            ],
+            [
+                '<?php ', // do not trim this as "<?php" is not valid PHP
+            ],
```

Comment:
can you add `don't fix` test for;
`"<?php\t\t"` two tabs
`"<?php  "` two spaces

or should those be trimmed down to `<?php ` single space .. .hmmm ;)  (I don't think these need fixing, but a test would be nice)

Reply:
I'd say trim them without the character that is part of `T_OPEN_TAG`.